# HG changeset patch
# User "Benoît Thébaudeau" <benoit.thebaudeau@advansee.com>
# Date 1323692711 -3600
# Node ID e98c6a720ce858a254ef3dbd4e5db5e245797706
# Parent  21f4f28e60ec0342133086c9a51e7f0e5b181fb8
cc/gcc: update Linaro GCC revisions to 2012.03

Update Linaro GCC with the latest available revisions.

Signed-off-by: "Benoît Thébaudeau" <benoit.thebaudeau@advansee.com>

diff --git a/config/cc/gcc.in b/config/cc/gcc.in
--- a/config/cc/gcc.in
+++ b/config/cc/gcc.in
@@ -36,9 +36,9 @@
 # Don't remove next line
 # CT_INSERT_VERSION_BELOW
 
-config CC_V_linaro_4_6_2011_10
+config CC_V_linaro_4_6_2012_03
     bool
-    prompt "linaro-4.6-2011.10 (EXPERIMENTAL)"
+    prompt "linaro-4.6-2012.03 (EXPERIMENTAL)"
     depends on CC_GCC_SHOW_LINARO
     select CC_GCC_4_6
 
@@ -60,9 +60,9 @@
     depends on EXPERIMENTAL
     select CC_GCC_4_6
 
-config CC_V_linaro_4_5_2011_10
+config CC_V_linaro_4_5_2012_03
     bool
-    prompt "linaro-4.5-2011.10 (EXPERIMENTAL)"
+    prompt "linaro-4.5-2012.03 (EXPERIMENTAL)"
     depends on CC_GCC_SHOW_LINARO
     select CC_GCC_4_5
 
@@ -325,10 +325,10 @@
     string
 # Don't remove next line
 # CT_INSERT_VERSION_STRING_BELOW
-    default "linaro-4.6-2011.10" if CC_V_linaro_4_6_2011_10
+    default "linaro-4.6-2012.03" if CC_V_linaro_4_6_2012_03
     default "4.6.1" if CC_V_4_6_1
     default "4.6.0" if CC_V_4_6_0
-    default "linaro-4.5-2011.10" if CC_V_linaro_4_5_2011_10
+    default "linaro-4.5-2012.03" if CC_V_linaro_4_5_2012_03
     default "4.5.3" if CC_V_4_5_3
     default "4.5.2" if CC_V_4_5_2
     default "4.5.1" if CC_V_4_5_1
